app/controllers/api/v0/user_groups_controller.rb

```diff
@@ -49,6 +49,7 @@ def index
             friendly_id: council.friendly_id,
             email: council.email,
           },
+          lead_user: council.reload.leader,
```

Comment:
This is because once the leader is changed, the `council.leader` still points to old leader. I didn't research much on why this is happening, because this anyway gets migrated to roles soon and this line of code won't be there.